cmake/external/firestore.cmake

```diff
@@ -49,6 +49,7 @@ function(GetTag t)
     DOWNLOAD_DIR ${FIREBASE_DOWNLOAD_DIR}
     GIT_REPOSITORY "https://github.com/firebase/firebase-ios-sdk.git"
     GIT_TAG ${t}
+    GIT_CONFIG "core.autocrlf=input"
```

Comment:
Just a heads-up that `GIT_CONFIG` was added in cmake 3.8, and we need to support cmake 3.6, since Android builds are currently pinned to cmake 3.6.

https://cmake.org/cmake/help/latest/module/ExternalProject.html

Reply:
Thanks..I think this is fine because Android does not get the dependency from git clone..

Reply:
Oh, but also we document that cmake 3.7 is the minimum supported version. So we shouldn't use cmake syntax that is not available in cmake 3.7.

https://github.com/firebase/firebase-cpp-sdk/blob/873dba22aabb9922d5574e91f626639eb364aaf7/README.md?plain=1#L49

In any case, it's fine for this experiment; but we'd need to do something different before merging anything into main.